includes/admin/shortcodes/shortcode-give-totals.php

```diff
@@ -72,26 +72,29 @@ public function define_fields() {
 			array(
 				'type' => 'container',
 				'html' => sprintf( '<p class="give-totals-shortcode-container-message">%s</p>',
-					 __( 'This shortcode shows the total amount raised towards a custom goal for one or several forms regardless of whether they have goals enabled or not.', 'give' )
+					__( 'This shortcode shows the total amount raised towards a custom goal for one or several forms regardless of whether they have goals enabled or not.', 'give' )
 				),
 			),
 			array(
 				'type' => 'container',
 				'html' => sprintf( '<p class="strong margin-top">%s</p>', __( 'Shortcode Configuration', 'give' ) ),
 			),
 			array(
-				'type'        => 'textbox',
-				'name'        => 'ids',
-				'label'       => __( 'Donation Form IDs:', 'give' ),
-				'tooltip'     => __( 'Enter the IDs separated by commas for the donation forms you would like to combine within the totals.', 'give' ),
+				'type'    => 'textbox',
+				'name'    => 'ids',
+				'label'   => __( 'Donation Form IDs:', 'give' ),
+				'tooltip' => __( 'Enter the IDs separated by commas for the donation forms you would like to combine within the totals.', 'give' ),
 			),
 			$category_lists,
 			$tag_lists,
 			array(
-				'type'    => 'textbox',
-				'name'    => 'total_goal',
-				'label'   => __( 'Total Goal:', 'give' ),
-				'tooltip' => __( 'Enter the total goal amount that you would like to display.', 'give' ),
+				'type'     => 'textbox',
+				'name'     => 'total_goal',
+				'label'    => __( 'Total Goal:', 'give' ),
+				'tooltip'  => __( 'Enter the total goal amount that you would like to display.', 'give' ),
+				'required' => array(
+					'alert' => esc_html__( 'You must first enter the total goal amount.', 'give' ),
```

Comment:
@DevinWalker Need you help to update this text

Reply:
@raftaar1191 We can add text like `Please enter a valid total goal amount`